.github/workflows/pr.yaml

```diff
@@ -91,18 +88,9 @@ jobs:
         with:
           version: "2023.1.3"
           install-go: false
-      - name: Check manifests # make generate manifests touches files even for no changes
```

Comment:
I am wondering why this step is deleted. Can it fix the issue?

Reply:
This step is not supported when we use "Ubuntu" directly.  When I try to make the "Check kubernetes manifests" task work,  I found it can't work well with using the "fedora:35" image. I am not sure if  the step ever work well before.

Reply:
What does it mean it's not supported? What kind of error did you get?

Reply:
I didn't save the error.  But from Johnny Berian, that way is not a supposed one.

Reply:
Who is Johnny Bieren?

Sorry but I don't understand what's going on here, I filed story to fix `Check kubernetes manifests` but I received:
* removal of fedora container and keeping default ubuntu, without explanation why (probably removal of task to install dependencies is related)
* the other task, which was working on fedora, is suddenly removed, without an explanation why, without showing what is the error
* no explanation in commit message 
* and task is fixed without mentioning what was the root cause

Reply:
Some idea for the fix is from referring other code with better practice.  I can't explain so much to you.